From 288f7e49f41f77319d2e10b4b6470a15205f039d Mon Sep 17 00:00:00 2001
From: Jason Chen <b02280@freescale.com>
Date: Fri, 1 Jul 2011 11:23:43 +0800
Subject: [PATCH 0251/2508] ENGR00152359-1 sii902x hdmi: add AVI and AIF
 support.

1.add AVI and AIF support.
2.add edid 4-block reading support.(not test)

Signed-off-by: Jason Chen <b02280@freescale.com>
(cherry picked from commit 449efb20625cc8a3dd1259796b4050578960a3d4)
---
 arch/arm/plat-mxc/include/mach/mxc_edid.h |    8 ++++++++
 1 files changed, 8 insertions(+), 0 deletions(-)

diff --git a/arch/arm/plat-mxc/include/mach/mxc_edid.h b/arch/arm/plat-mxc/include/mach/mxc_edid.h
index f354461..ec65a5f 100755
--- a/arch/arm/plat-mxc/include/mach/mxc_edid.h
+++ b/arch/arm/plat-mxc/include/mach/mxc_edid.h
@@ -32,8 +32,16 @@ struct mxc_edid_cfg {
 	bool cea_ycbcr444;
 	bool cea_ycbcr422;
 	bool hdmi_cap;
+
+	/*VSD*/
+	bool vsd_dc_48bit;
+	bool vsd_dc_36bit;
+	bool vsd_dc_30bit;
+	bool vsd_dc_y444;
+	bool vsd_dvi_dual;
 };
 
+int mxc_edid_var_to_vic(struct fb_var_screeninfo *var);
 int mxc_edid_read(struct i2c_adapter *adp, unsigned short addr,
 	unsigned char *edid, struct mxc_edid_cfg *cfg, struct fb_info *fbi);
 
-- 
1.7.7.4

